tests/data/db/KEYREGISTRY

```diff
@@ -0,0 +1 @@
+�-P��Pg����(�=Hello Badger
```

Comment:
please remove this and other tests/data files!

Reply:
This is needed for `TestPruneState`. Otherwise generating 256 blocks in unit test by spinning up nodes takes time.